db/schema.rb

```diff
@@ -57,6 +56,7 @@
     t.string   "media_url"
     t.boolean  "visible_content"
     t.boolean  "visible_image"
+    t.boolean  "media_landscape", default: false
```

Comment:
Ah, I misunderstood the PR title.  Thought it was for news messages/ news paper.  Will update the issue to clarify.